garden-backend-service/src/models/_associations.py

```diff
@@ -19,8 +19,9 @@
 users_saved_gardens = Table(
     "users_saved_gardens",
     Base.metadata,
-    Column("user_id", Integer, ForeignKey("users.id"), primary_key=True),
-    Column("garden_id", Integer, ForeignKey("gardens.id"), primary_key=True),
+    Column("user_id", Integer, ForeignKey("users.id")),
+    Column("garden_id", Integer, ForeignKey("gardens.id")),
+    PrimaryKeyConstraint("user_id", "garden_id"),
```

Comment:
Note: this did not "change" things from Alembic's perspective. I had to write the migration manually.